website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -291,6 +291,15 @@ var QuickSearchProject = {
             return self.preSelectDirection() === sort ? 'selected' : 'not-selected';
         };
 
+        // Filtering on root project
+        self.rootMatch = function (node) {
+            try {
+                var root = project.embeds.root.data.attributes.title;
+                return (node.embeds.root.data.attributes.title.toUpperCase().indexOf(self.filter().toUpperCase()) !== -1);
```

Comment:
Rather than doing a try catch, you can use `lodash.get` to get a deeply nested attribute.

```js
var lodashGet = require('lodash.get');
//..
var rootTitle = lodashGet(node, 'embeds.root.data.attributes.title') || '';
return rootTitle.toUpperCase()....
```